tests/test_multiresolution.py

```diff
@@ -0,0 +1,80 @@
+import warnings
+warnings.filterwarnings('ignore')
+
+import numpy as np
+from numpy.testing import assert_allclose
+import scarlet2
+
+import numpy as np
+import astropy.io.fits as fits
+from astropy.wcs import WCS
+
+import jax.numpy as jnp
+
+# Load the HSC image data
+obs_hdu = fits.open('../scarlet-test-data/data/test_resampling/Cut_HSC1.fits')
```

Comment:
Can we pull this file off github without getting the repo in this particular directory? FITS is technically a text-base file format, so what happens if we try to get the raw file with a URL request?